src/atoms/breadcrumbs.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {Breadcrumbs as MuiBreadcrumbs, BreadcrumbsProps as MuiBreadcrumbsProps} from '@mui/material';
+import Link from '@mui/material/Link';
+
+interface NavigationItem {
+    label?: string;
+    path?: string;
+}
+
+interface BreadcrumbProps extends MuiBreadcrumbsProps {
+    separator?: string | JSX.Element;
+    items: NavigationItem[];
+}
+
+function BreadCrumbs({ separator, items }: BreadcrumbProps) {
```

Comment:
Ingeneer, i think it would be a good idea if you add separator default value ">",  since that is what we want to use in case no value is enter.
what do you think?